app/telemetry/handler.py

```diff
@@ -24,6 +24,10 @@ def handle_message(event, context):
         for tile in telemetry["tiles"]:
             if "shim" in tile:
                 ping_adzerk(tile['shim'])
+    elif "metrics" in telemetry:
```

Comment:
The ping, [as documented](https://dictionary.telemetry.mozilla.org/apps/fenix/pings/spoc), is sent for both clicks and impressions. We should check `ping_info.reason` to see what this ping is about and bubble the information accordingly as needed.

Reply:
I don't know what is meant here by "bubble the information accordingly".

I suspect you mean for it to be added to the value passed to ping_adzerk. That function is being passed a checksummed string, so it's unclear to me how that would be done, or even what the string contains, and I assumed that pocket was formatting the value in the client with all necessary information.

Reply:
I mean that this should be something like

```suggestion
    elif ("ping_info" in telemetry") and ("metrics" in telemetry):
        # reason is either a 'click' or an 'impression'. 
        reason = ping_info.reason
```

If adzerk is only expecting 'impressons', then we should not submit the SHIM in case this is a 'click' (user clicked on the ad).

Reply:
@Dexterp37, Kevel (formerly AdZerk) are expecting shims for both clicks and impressions.

Reply:
@Dexterp37, from what I know, the shim contains all the info Kevel needs, including whether it's an impression or a click, so we should be good there sending just the shim as we do from the desktop browser.